runtime/oti/cfr.h

```diff
@@ -856,6 +856,7 @@ typedef struct J9CfrClassFile {
 #define CFR_ACC_SYNCHRONIZED  				0x00000020
 #define CFR_ACC_BRIDGE  					0x00000040
 #define CFR_ACC_VOLATILE  					0x00000040
+#define CFR_ACC_ATOMIC						0x00000040
```

Comment:
May as well format this like all the other lines.

Reply:
Format fixed.